pypeit/core/coadd.py

```diff
@@ -2504,6 +2557,10 @@ def ech_combspec(waves, fluxes, ivars, masks, sensfile, nbest=None, wave_method=
               stacked spectrum on wave_stack wavelength grid. True=Good.
     """
 
+#        merge_stack: bool, default=False,
```

Comment:
remove?

Reply:
Please leave this code here for the time being.

Reply:
Added some comments to ask developers to leave this in.